src/generators/crud/default/views/index.php

```diff
@@ -208,7 +208,7 @@
                         'aria-label' => Yii::t('{$generator->messageCategory}', 'View'),
                         'data-pjax' => '0',
                     ];
-                    return Html::a('<span class="glyphicon glyphicon-file"></span>', \$url, \$options);
+                    return Html::a('<span class="glyphicon glyphicon-eye-open"></span>', \$url, \$options);
```

Comment:
I do not favor changing the icons, CC: @eluhr WDYT?

Reply:
I think an open eye describes the target page better than a file icon. I'd change it.

Reply:
OK :smile:

Reply:
On index.php CRUD view use file icon, but in view.php use eye icon. Change to the same style as used in the framework gii generated CRUDs